  File "/opt/skynet/bin/skynetd.py", line 224
    double_streamer("HVAC_HystTemp","%.1f" % hyst_temp)
                  ^
SyntaxError: invalid syntax
  File "/opt/skynet/bin/skynetd.py", line 224
    double_streamer("HVAC_HystTemp","%.1f" % hyst_temp)
                  ^
SyntaxError: invalid syntax
  File "/opt/skynet/bin/skynetd.py", line 731
    text_file.write(str(HEAT_times[0]) + "\n)
                                            ^
SyntaxError: EOL while scanning string literal
  File "/opt/skynet/bin/skynetd.py", line 612
    if ((timeNow - HEAT_times[0]) > hyst_time) or ((timeNow - startTime) > 60 and (((timeNow - startTime) < hyst_time) and startupburn = False)) or (HVAC_status[2] == 1):
                                                                                                                                       ^
SyntaxError: invalid syntax
/opt/skynet/bin/skynetd.py:619: SyntaxWarning: name 'startupburn' is used prior to global declaration
  global startupburn
/opt/skynet/bin/skynetd.py:620: SyntaxWarning: name 'startupburn' is used prior to global declaration
  global startupburn
  File "/opt/skynet/bin/skynetd.py", line 255
    double_streamer("secondsSinceStartup",seconds_since_startup)
                  ^
SyntaxError: invalid syntax
/opt/skynet/bin/skynetd.py:622: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:622: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:627: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:632: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:632: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:632: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:629: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:629: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:630: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:642: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:601: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:600: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:600: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:600: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:600: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
/opt/skynet/bin/skynetd.py:600: SyntaxWarning: name 'startupBurn' is used prior to global declaration
  global startupBurn
Connected with result code 0
sensors/centipede01/temperature 24.87
Traceback (most recent call last):
  File "./mqtt-client.py", line 29, in <module>
    client.loop_forever()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 1378, in loop_forever
    rc = self.loop(timeout, max_packets)
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 897, in loop
    rc = self.loop_read(max_packets)
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 1177, in loop_read
    rc = self._packet_read()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 1766, in _packet_read
    rc = self._packet_handle()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 2239, in _packet_handle
    return self._handle_publish()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 2414, in _handle_publish
    self._handle_on_message(message)
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 2574, in _handle_on_message
    self.on_message(self, self._userdata, message)
  File "./mqtt-client.py", line 17, in on_message
    f.writeln(msg.topic+" "+str(msg.payload))
AttributeError: 'file' object has no attribute 'writeln'
Connected with result code 0
Traceback (most recent call last):
  File "./mqtt-client.py", line 33, in <module>
    client.loop_forever()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 1378, in loop_forever
    rc = self.loop(timeout, max_packets)
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 897, in loop
    rc = self.loop_read(max_packets)
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 1177, in loop_read
    rc = self._packet_read()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 1766, in _packet_read
    rc = self._packet_handle()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 2239, in _packet_handle
    return self._handle_publish()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 2414, in _handle_publish
    self._handle_on_message(message)
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 2574, in _handle_on_message
    self.on_message(self, self._userdata, message)
  File "./mqtt-client.py", line 20, in on_message
    f = open("/var/log/skynet/"+date.isoformat()+"-mqtt.log", "a")
NameError: global name 'date' is not defined
Connected with result code 0
Traceback (most recent call last):
  File "./mqtt-client.py", line 33, in <module>
    client.loop_forever()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 1378, in loop_forever
    rc = self.loop(timeout, max_packets)
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 897, in loop
    rc = self.loop_read(max_packets)
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 1177, in loop_read
    rc = self._packet_read()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 1766, in _packet_read
    rc = self._packet_handle()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 2239, in _packet_handle
    return self._handle_publish()
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 2414, in _handle_publish
    self._handle_on_message(message)
  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 2574, in _handle_on_message
    self.on_message(self, self._userdata, message)
  File "./mqtt-client.py", line 20, in on_message
    f = open("/var/log/skynet/"+date.isoformat()+"-mqtt.log", "a")
TypeError: descriptor 'isoformat' of 'datetime.date' object needs an argument
